test/core_vnode_eqc.erl

```diff
@@ -52,17 +52,25 @@ simple_test_() ->
     {setup,
      fun setup_simple/0,
      fun(OldVars) ->
-             riak_core_ring_manager:stop(),
-	     application:stop(exometer),
-	     application:stop(lager),
-	     application:stop(goldrush),
-             [ok = application:set_env(riak_core, K, V) || {K,V} <- OldVars],
-             ok
+         unlink(whereis(riak_core_ring_manager)),
```

Comment:
How about adding a util function, say `unlink_named(ProcessName)`, and use that to make the code uniform?